src/main/java/org/traccar/MainModule.java

```diff
@@ -375,6 +376,12 @@ public static DriverEventHandler provideDriverEventHandler(IdentityManager ident
         return new DriverEventHandler(identityManager);
     }
 
+    @Singleton
+    @Provides
+    public static Timer provideGlobalTimer() {
```

Comment:
Please rename to `provideTimer`.

Reply:
Ok. I see your point. Now, do you think it's ok to use the class `Timer` for this purpose?

Reply:
I think it makes sense.

Reply:
Ok then.